internal/gather/cni.go

```diff
@@ -172,7 +172,7 @@ func gatherOVNResources(info *Info, networkPlugin string) {
 
 func gatherCableDriverResources(info *Info, cableDriver string) {
 	logPodInfo(info, "cable driver data", gatewayPodLabel, func(info *Info, pod *v1.Pod) {
-		if cableDriver == libreswan {
+		if cableDriver == libreswan || cableDriver == "" { // If none specified, use libreswan as default
```

Comment:
Instead of hard-coding Libreswan as the default, ask the cable drivers what the default is:

```suggestion
		if cableDriver == libreswan || (cableDriver == "" && cable.GetDefaultCableDriver() == libreswan) {
```

Reply:
Don't think this will work. `GetDefaultCableDriver()` fetches value from a global variable that is set in submariner engine. It won't be available in subctl

Reply:
Ah yes, you’re right!